MapboxAndroidDemo/src/main/res/values/descriptions_strings.xml

```diff
@@ -8,6 +8,7 @@
     <string name="activity_basic_support_map_frag_description">Include a map fragment within your app using Android support library.</string>
     <string name="activity_basic_mapbox_options_description">Add a mapview in a dynamically created layout.</string>
     <string name="activity_basic_mapbox_kotlin_description">Display a basic map with Kotlin code.</string>
+    <string name="activity_basic_kotlin_support_map_frag_description">Display a basic map with Kotlin code.</string>
```

Comment:
Should this string be consistent with activity_basic_support_map_frag_description?